api/test/integration/test_experimental_endpoints.tavern.yaml

```diff
@@ -1,9 +1,87 @@
 ---
-test_name: DELETE /experimental/syscheck
+test_name: DELETE /experimental/rootcheck
 
 marks:
   - base_tests
 
+stages:
+
+  # DELETE /experimental/rootcheck
+  - name: Delete rootcheck scans in agent 001
+    request:
+      verify: False
+      method: DELETE
+      url: "{protocol:s}://{host:s}:{port:d}/experimental/rootcheck"
+      headers:
+        Authorization: "Bearer {test_login_token}"
+      params:
+        agents_list: "001"
+    response:
+      status_code: 200
+      json:
+        error: 0
+        data:
+          affected_items:
+            - "001"
+          failed_items: []
+          total_affected_items: 1
+          total_failed_items: 0
+
+  # DELETE /experimental/rootcheck
+  - name: Delete rootcheck scans in all agents
+    request:
+      verify: False
+      method: DELETE
+      url: "{protocol:s}://{host:s}:{port:d}/experimental/rootcheck"
+      headers:
+        Authorization: "Bearer {test_login_token}"
+      params:
+        agents_list: all
+    response:
+      status_code: 200
+      json:
+        error: 0
+        data:
+          affected_items:
+            - "000"
+            - "001"
+            - "002"
+            - "003"
+            - "004"
+            - "005"
+            - "006"
+            - "007"
+            - "008"
+            - "009"
+            - "010"
+            - "011"
+            - "012"
+          failed_items: []
+          total_affected_items: 13
+          total_failed_items: 0
+
+  # GET /rootcheck/002
```

Comment:
Remove this test, it may cause race conditions with the DB

Reply:
Done